lib/reporter.js

```diff
@@ -225,6 +226,13 @@ class CucumberReporter {
         })
     }
 
+    getTitle (featureOrScenario) {
+        const name = featureOrScenario.getName()
+        const tags = featureOrScenario.getTags ? featureOrScenario.getTags() : []
```

Comment:
if `getTags` is only available for Scenarios why do we accept features or scenarios here? In which cases is getTags undefined?